app/src/main/java/com/nutomic/syncthingandroid/views/FoldersAdapter.java

```diff
@@ -102,12 +102,12 @@ public View getView(int position, View convertView, @NonNull ViewGroup parent) {
             }
             binding.items.setVisibility(VISIBLE);
             binding.items.setText(getContext().getResources()
-                    .getQuantityString(R.plurals.files, (int) model.inSyncFiles, model.inSyncFiles, model.globalFiles));
+                    .getQuantityString(R.plurals.files, (int) FolderStatusInfo.inSyncFiles, FolderStatusInfo.inSyncFiles, FolderStatusInfo.globalFiles));
             binding.size.setVisibility(VISIBLE);
             binding.size.setText(getContext().getString(R.string.folder_size_format,
-                    Util.readableFileSize(getContext(), model.inSyncBytes),
-                    Util.readableFileSize(getContext(), model.globalBytes)));
-            setTextOrHide(binding.invalid, model.invalid);
+                    Util.readableFileSize(getContext(), FolderStatusInfo.inSyncBytes),
+                    Util.readableFileSize(getContext(), FolderStatusInfo.globalBytes)));
+            setTextOrHide(binding.invalid, FolderStatusInfo.invalid);
```

Comment:
Just realized that this method is really long. It would be a good idea to split it up somehow (but not necessarily in this PR).